### PR TITLE
getPricesV2.js Added.

### PR DESCRIPTION
**Pull request by harrygcollins**
getPricesV2.js added to allow for the client to access and get values from the database.

### CODE REVIEW

**Comment by LoicVerrall** [comment]
@harrygcollins, do you think it will be relatively straightforward for you to extract functionality out of the main function into individual mini functions to make Code Climate calm down? 😜

File: getPricesV2.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
+exports.handler = (event, context, callback) => {
+    // Setup
+    var AWS = require("aws-sdk");
+    var docClient = new AWS.DynamoDB.DocumentClient();
```
Comment (LoicVerrall) [inline getPricesV2.js]: Typically these two 'import' statements go _outside_ the handler function

File: getPricesV2.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+ 
+    // User Inputs. 
+    var latitude = event.latitude;
+    var longitude = event.longitude;
+    var distance = event.distance;
```
Comment (LoicVerrall) [inline getPricesV2.js]: Shouldn't we be accessing the parameters using the `event.queryStringParameters.key1` kind of syntax?

As we're accepting HTTP POST requests from the client.

File: getPricesV2.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+             ":lonFrom" :   Math.min(start_lon, end_lon),
+             ":lonTo"   :   Math.max(start_lon, end_lon)
+        }
+    };
+ 
+    //List of items that will be returned. Needed to account for DynamoDB window size. 
+    var items = []
+    
+    // Accounts for DynamoDB only being able to send a set window at a time, scans whole DB. 
+    var scanExecute = function(callback) {
```
Comment (LoicVerrall) [inline getPricesV2.js]: I think this should be extracted into its own separate function (outside the `exports.handler` function)

**Review by LoicVerrall — DISMISSED** [review]
The 'cognitive complexity' of this program is high according to Code Climate.

To reduce it, I've suggested some changes (like extract that inner function).

Looks good though!

**Comment by harrygcollins** [comment]
Changed the store location to a new file and changed my file to index to allow for CI.

**Comment by harrygcollins** [comment]
Added the Lat/Lon regex method.

**Comment by harrygcollins** [comment]
@Tang-1996 I have had to change the index file to allow travis to work. Can you change your tests to allow travis to work? If you comment out the User ID tests but the Lat/Lon ones should still be relevant to this.

**Pull request merged**